### PR TITLE
Updating GKE module.

### PR DESCRIPTION
**Pull request by soumyapani**

### CODE REVIEW

File: terraform/modules/cluster/mig/main.tf [hunk 1]
```diff
@@ -47,7 +47,7 @@ module "dashboard" {
   source = "../../common/dashboard"
   count  = var.enable_ops_agent ? 1 : 0
 
-  enable_gce_gke_gpu_utilization_widgets = true
+  enable_gce_gke_gpu_utilization_widgets = false
```
Comment (stevenBorisko) [inline terraform/modules/cluster/mig/main.tf:50]: good catch!

File: test/terraform/modules/cluster/gke/input/gke-nongpu.tfvars [hunk 2]
```diff
@@ -0,0 +1,11 @@
+project_id      = "gce-ai-infra"
+resource_prefix = "ci"
+region          = "us-central1"
+node_pools = [{
+  zone                     = "us-central1-a"
+  node_count               = 1
+  machine_type             = "n1-standard-1"
+  guest_accelerator_type   = ""
+  guest_accelerator_count  = 0
```
Comment (stevenBorisko) [inline test/terraform/modules/cluster/gke/input/gke-nongpu.tfvars:9]: Do you think this should be preferred over `guest_accelerator { type = string, count = number}` (and here `guest_accelerator = null`)?

**Review by stevenBorisko — COMMENTED** [review]
just one question because we should probably make it the same across all cluster no matter which one we choose. I think it would make more sense to do it the object way rather than splitting since there is never a scenario where we would want to specify one and not the other.

File: test/terraform/modules/cluster/gke/input/gke-nongpu.tfvars (hunk 2 above)
Comment (soumyapani) [reply test/terraform/modules/cluster/gke/input/gke-nongpu.tfvars]: addressed in latest iteration.

**Review by stevenBorisko — APPROVED** [review]

**Pull request merged**